src/common/components/Profile/StarRate.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
```

Comment:
@reacteer I think we need to rebuild the star rate component again as it's hardcoded.

Reply:
@dbizhev , I will ask design team for empty star icon.